Snippets/ServiceFabricPersistence/ServiceFabricPersistence_1/SynchronizedStorageSession.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace ServiceFabricPersistence_1
+{
+    using System.Threading.Tasks;
+    using Microsoft.ServiceFabric.Data.Collections;
+    using NServiceBus;
+    using NServiceBus.Persistence.ServiceFabric;
+
+    public class SynchronizedStorageSession
+    {
+        class Message : IMessage { }
+
+        #region ServiceFabricPersistenceSynchronizedSession
+        class Handler : IHandleMessages<Message>
```

Comment:
Do we need a using in saga section as https://docs.particular.net/nservicebus/sql-persistence/#sqlstoragesession including a note?